services/frontend-v3/src/components/layouts/landingLayout/LandingLayout.jsx

```diff
@@ -1,13 +1,31 @@
 import React from "react";
 import LandingLayoutView from "./LandingLayoutView";
+import backgroundOptionOne from "../../../assets/landing-1.svg";
+import backgroundOptionTwo from "../../../assets/landing-2.svg";
+import backgroundOptionThree from "../../../assets/landing-3.svg";
 
 /**
  *  LandingLayout(props)
  *
  *  this is the controller for LandingLayoutView
  */
 const LandingLayout = () => {
-  return <LandingLayoutView />;
+  /**
+   * Random Picture Select
+   *
+   * select a random picture for landing page
+   */
+  const randomPictureSelect = () => {
+    const imageOptions = [
+      backgroundOptionOne,
+      backgroundOptionTwo,
```

Comment:
just wondering why are we selecting a random image for the background?

Reply:
honestly it was just a little fun thing we did cause we had a few left over designs. i can always remove it if its causing issues